cmd/telemeter-server/main.go

```diff
@@ -100,7 +99,7 @@ func main() {
 	cmd.Flags().StringVar(&opt.TLSKeyPath, "tls-key", opt.TLSKeyPath, "Path to a private key to serve TLS for external traffic.")
 	cmd.Flags().StringVar(&opt.TLSCertificatePath, "tls-crt", opt.TLSCertificatePath, "Path to a certificate to serve TLS for external traffic.")
 
-	cmd.Flags().StringVar(&opt.StorageDir, "storage-dir", opt.StorageDir, "The directory to persist incoming metrics. If not specified metrics will only live in memory.")
+	cmd.Flags().StringVar(&opt.StorageDir, "storage-dir", opt.StorageDir, "DEPRECATED: The directory to persist incoming metrics. If not specified metrics will only live in memory.")
```

Comment:
I don't think we need to worry about breaking people. No one is using telemeter yet. This option has no utility in the code any more, so let's simply remove it.

Reply:
sounds good :+1: